src/app/components/applications/SSV/MyAccount/components/ClusterDashboard/ClusterDashboard.tsx

```diff
@@ -120,7 +130,7 @@ const ClusterDashboard = () => {
         <Grid container item xs className={classes.HeaderButtonsWrapper}>
           {rows.length > 0 && (<Grid item className={`${classes.HeaderButton} ${classes.lightHeaderButton}`} onClick={moveToFeeRecipient}>
             Fee Address
-            <Grid item className={classes.Pencil}/>
+            {loadingFeeRecipient ? <CircularProgress className={classes.SpinnerWrapper} thickness={6} size={16} /> : <Grid item className={classes.Pencil}/>}
           </Grid>)}
           <Grid item className={classes.HeaderButton} onClick={moveToRegisterValidator}>Add Cluster</Grid>
```

Comment:
Should you prevent another click during `loadingFeeRecipient` ?
like: onClick={() => { loadingFeeRecipient && moveToRegisterValidator(); }}